eslintrc.json

```diff
@@ -561,27 +561,47 @@
     {
       "files": "*.vue",
       "globals": {
-        "apos": true,
-        "defineProps": "readable",
-        "defineEmits": "readable",
-        "defineExpose": "readable",
-        "defineOptions": "readable",
-        "defineModel": "readable",
-        "defineSlots": "readable"
+        "apos": "readonly",
+        "defineProps": "readonly",
+        "defineEmits": "readonly",
+        "defineExpose": "readonly",
+        "defineOptions": "readonly",
+        "defineModel": "readonly",
+        "defineSlots": "readonly",
+        "location": "readonly",
+        "history": "readonly",
+        "sessionStorage": "readonly",
+        "localStorage": "readonly",
+        "IntersectionObserver": "readonly",
+        "Image": "readonly",
+        "Node": "readonly",
+        "ResizeObserver": "readonly",
+        "addEventListener": "readonly",
+        "removeEventListener": "readonly"
```

Comment:
I thought that was the whole point behind adding
```json
"env": {
  "browser": true
}
```
Is something wrong with our setup?

Reply:
Oooh nice, didn't know that. You right, I had to add it for bue files because we didn't specify `env: browser`. Thanks removing it.